tests/integration/pilot/mcp/testdata/dataset.go

```diff
@@ -0,0 +1,17 @@
+//  Copyright 2018 Istio Authors
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+package testdata
+
+//go:generate $GOPATH/src/istio.io/istio/bin/go-bindata.sh --nocompress --nometadata --pkg testdata -o dataset.gen.go dataset/...
```

Comment:
Is there any reason that you have to go through all of this with the test data?  I think you copied it from Galley but I think it's over-kill. Can you just use pre-defined k8s deployments (yaml) which you know will result in particular SSEs, and then use structpath library expressions to check a few fields?

Reply:
sure that was on my todo list, it's done.